src/site/components/navigation-links-list.html~9bf7b9b4ee81ded6b7187f3c13b35ce2d81b9884

```diff
@@ -0,0 +1,71 @@
+{% comment %}
```

Comment:
Should this file have a commit hash at the end of its name?

Reply:
@rianfowler : This is a file that was made when I pulled in master.
![screen shot 2018-11-20 at 8 15 01 am](https://user-images.githubusercontent.com/41440372/48775912-69ddd880-ec9c-11e8-8d92-dcbd2b2f7934.png)

Can these two files be deleted?

Reply:
@rianfowler : removed. Thank you!